elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryThread.java

```diff
@@ -49,10 +53,13 @@ public AsyncAPIResult call() throws URISyntaxException, NoHttpResponseException
         ElideResponse response = null;
         log.debug("AsyncQuery Object from request: {}", this);
         UUID requestUUID = UUID.fromString(queryObj.getRequestId());
+        Map<String, List<String>> bypassCacheMap = new HashMap<String, List<String>>();
+        bypassCacheMap.put("bypasscache", Arrays.asList(String.valueOf(service.isBypassCache())));
```

Comment:
Won't this always override the cache?  I don't think this is the behavior that we want.